### ISSUE TITLE
"Missing `size` Parameter": Build Failure on Flutter Beta 3.18.0-0.2.pre due to `ViewConfiguration` API Change

### ISSUE THREAD

**Issue by lukemmtt** [ABausG/home_widget#199]
Issue Description:
The `home_widget` package currently fails to compile when using the Flutter Beta 3.18.0-0.2.pre due to a breaking change in the `ViewConfiguration` class. This change may impact the package's compatibility if it persists in the stable release.

Error Output:
```
Error: No named parameter with the name 'size'.
          size: logicalSize,
          ^^^^
```

Steps to Reproduce:
1. Switch to the Flutter beta channel (`flutter channel beta`).
2. Upgrade to Flutter version 3.18.0-0.2.pre (`flutter upgrade`).
3. Attempt to compile a project with the `home_widget` package.

Flutter Environment:
- Flutter SDK Version: 3.18.0-0.2.pre (Beta Channel)
- `home_widget` package version: 0.4.1

Note:
This issue is documented to acknowledge the current incompatibility with the beta channel and to track any future updates or changes that may affect the package's functionality. It is similar to the issue faced by the `screenshot` package, detailed in [screenshot issue #158](https://github.com/SachinGanesh/screenshot/pull/158). There is no immediate expectation for a fix, as the beta channel changes are not guaranteed to be part of the stable release.

**Comment by derekhskim** [comment]
I've encountered the same issue and I think Flutter 3.19.0-0.1 release had a roll-back to take the size parameter again, so when I've upgraded the flutter to most recent version, it worked for me.

**Comment by andreidevo** [comment]
nah, doesn't working. even with upgrading. 
So you need to Copy this Dev and make it changeable

**Comment by esDotDev** [comment]
It now compiles on Beta, but not on Master.

**Comment by PetrKubes97** [comment]
@esDotDev Which Beta version are you using? Fails on 3.20.0-1.1.pre.

**Comment by ABausG** [comment]
My guess when @esDotDev wrote it, Beta was still on 3.19 which is now Stable. Guess 3.20 removes the value.

**Comment by ABausG** [comment]
This is fixed in Version 0.6.0 which requires 3.20
